Documentation/Storage-Configuration/Ceph-CSI/custom-images.md

```diff
@@ -27,6 +27,12 @@ ROOK_CSI_SNAPSHOTTER_IMAGE: "registry.k8s.io/sig-storage/csi-snapshotter:v6.2.1"
 ROOK_CSIADDONS_IMAGE: "quay.io/csiaddons/k8s-sidecar:v0.5.0"
 ```
 
+!!! note
```

Comment:
Instead of a note how about a header so it's obvious this feature is for using a private repo?
```suggestion
### **Use private repo**
```